From: Piotr Luc <piotr.luc@intel.com>
Date: Wed Oct 12 20:27:25 2016 +0200
Subject: perf/x86/intel/rapl: Add Knights Mill CPUID
Patch-mainline: v4.9-rc2
Git-commit: 36c4b6c14d20b37fda79cbcd3e8ef7d11f5ef9dc
References: FATE#321923
Signed-off-by: Tony Jones <tonyj@suse.de>

 [SUSE: adjust for:
  no 609d809f
    perf/x86: Move perf_event_intel_rapl.c ....... => x86/events/intel/rapl.c
  no 970442c5
    x86/cpu/intel: Introduce macros for Intel family numbers
 ]

---
 arch/x86/kernel/cpu/perf_event_intel_rapl.c | 1 +
 1 file changed, 1 insertion(+)
diff --git a/arch/x86/kernel/cpu/perf_event_intel_rapl.c b/arch/x86/kernel/cpu/perf_event_intel_rapl.c
index ed446bd..ffc10ab 100644
--- a/arch/x86/kernel/cpu/perf_event_intel_rapl.c
+++ b/arch/x86/kernel/cpu/perf_event_intel_rapl.c
@@ -741,6 +741,7 @@ static int __init rapl_pmu_init(void)
 		rapl_pmu_events_group.attrs = rapl_events_srv_attr;
 		break;
 	case 87: /* Knights Landing */
+	case 133: /* Knights Mill */
 		rapl_add_quirk(rapl_hsw_server_quirk);
 		rapl_cntr_mask = RAPL_IDX_KNL;
 		rapl_pmu_events_group.attrs = rapl_events_knl_attr;

